pyscriptjs/src/runtime.ts

```diff
@@ -171,6 +172,8 @@ export abstract class Runtime extends Object {
             loader?.close();
         }
 
+        createCustomElements();
+
```

Comment:
I still don't understand, why you want to create custom elements *afer* closing the splashscreen?
My gut feeling is that we want to create them before, and close the splashscreen only after everything has been created. Am I missing anything?

Reply:
No you are right it makes no sense doing it this way 😄  I've moved this up so it runs before we close the splashscreen